docs/data/material/customization/color/ColorDemo.js

```diff
@@ -58,7 +58,7 @@ function ColorDemo(props) {
             </Typography>
           </Toolbar>
         </AppBar>
-        <Box component="pre" sx={{ m: 2, fontSize: 16 }}>
+        <Box component="pre" sx={{ m: 2, fontSize: 16, overflowX: 'scroll' }}>
```

Comment:
In the [demo](https://deploy-preview-43672--material-ui.netlify.app/material-ui/customization/color/#playground), even if there is no need to scroll, the scrollbar is rendered. We can show scrollbar only when needed:
```suggestion
        <Box component="pre" sx={{ m: 2, fontSize: 16, overflowX: 'auto' }}>
```